ghost/config/default.cfg

```diff
@@ -460,6 +460,9 @@ oh_votemode_time = 30
 oh_cname = grief-code 00FF00Grief
 oh1_cname = neubivljiv 0000FFNeub
 
+###
+oh_hiddenColoredName = 1

```

Comment:
Variable name does not fit to its functionality.
Rename **oh_hiddenColoredName** to **oh_ColoredNames** in all files and i will merge your request.
Names never get hidden.. they get colored or they dont.